README.md

```diff
@@ -9,6 +9,12 @@ Collection of serverless API to enable public access to data surrounding the Bad
 Interactive Swagger Documentation  
 https://docs.badger.finance/
 
+## Docs:
+
+- [Api v2](./docs/api-v2.md)
```

Comment:
do you mind making this one `API v2` as a minor nit, and if we can replace the link above with https://api.badger.com/docs/